src/types.jl

```diff
@@ -144,7 +144,7 @@ immutable HSV{T<:AbstractFloat} <: Color{T,3}
 end
 
 "`HSB` (Hue-Saturation-Brightness) is an alias for `HSV`."
-HSB(h, s, b) = HSV(h, s, b)
+typealias HSB HSV
```

Comment:
Was there a reason to not do a typealias that I did not consider? This line was untested, so I took the easy way :-)